### PR TITLE
Fixes Grills

### PR DESCRIPTION
**Pull request by dwasint**
# About The Pull Request
https://github.com/tgstation/tgstation/pull/65833

## Why It's Good For The Game
Gotta grill

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog

:cl:
fixes: grills now work again
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/kitchen_machinery/grill.dm [hunk 1]
```diff
@@ -18,57 +19,67 @@
...
+			to_chat(user, span_notice("You pour the Monkey Energy in [src]."))
+			I.reagents.remove_reagent(/datum/reagent/consumable/monkey_energy, I.reagents.get_reagent_amount(/datum/reagent/consumable/monkey_energy))
+			update_appearance()
+			return
+	else if(IS_EDIBLE(I))
+		if(HAS_TRAIT(I, TRAIT_NODROP) || (I.item_flags & (ABSTRACT | DROPDEL)))
+			return ..()
+		else if(HAS_TRAIT(I, TRAIT_FOOD_GRILLED))
+			to_chat(user, span_notice("[I] has already been grilled!"))
+			return
```
Comment (nednaZ) [inline code/modules/food_and_drinks/kitchen_machinery/grill.dm:65]: Remove this, since it won't matter if you grill something repeatedly

**Review by nednaZ — CHANGES_REQUESTED** [review]
After testing it, it seems like the original smoke creation and sound no longer exists.
There is also no way to check if it's done or not unless you take an item off the grill, which makes it impossible to put it back on the grill afterwards.

**Comment by dwasint** [comment]
> After testing it, it seems like the original smoke creation and sound no longer exists. There is also no way to check if it's done or not unless you take an item off the grill, which makes it impossible to put it back on the grill afterwards.

smoke should still work i never touched that part of the code? as for sound no idea why it broke reverted it back however

**Review by nednaZ — APPROVED** [review]
Looks like it's working as intended

**Pull request merged**